src/components/Button.jsx

```diff
@@ -1,9 +1,9 @@
 import Link from "next/link";
 
-const Button = ({ link, text, target }) => {
+const Button = ({ link, text, target, color }) => {
```

Comment:
```suggestion
const Button = ({ link, text, target, color="text-black" }) => {
```

you can add a default prop if nothing is passed into color